resources/views/fontend/contract/contract.blade.php

```diff
@@ -0,0 +1,57 @@
+@extends('fontend.layouts.master')
+@section('content')
+<!-- User page start -->
+<div class="user-page content-area-7">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="search-area">
+                    <div class="search-area-inner">
+                        <div class="search-contents ">
+                            @if (count($errors) > 0)
+                            <div class="alert alert-danger">
+                                @foreach ($errors->all() as $err)
+                                {{ $err }}<br>
+                                @endforeach
+                            </div>
+                            @endif
+                            @if (session('message'))
+                            <div class="alert alert-success">
+                                {{ session('message') }}
+                            </div>
+                            @endif
+                            {!! Form::open(['method' => 'POST', route('contract', $ct->id )]) !!}
+                            <h3 class="heading">{{ __('label.contract') }}</h3>
+                            <div class="row">
+                                <div class="col-lg-12 ">
+                                    <div class="form-group name">
+                                        {!! Form::label('time', trans('message.startday')) !!}<br>
+                                        {!! Form::date('date') !!}
+                                    </div>
+                                    <div class="form-group name">
+                                        {!! Form::label('content', trans('message.time')) !!}
+                                        {!! Form::text('time', null, ['class' => 'form-control', 'placeholder' => __('label.property_title')]) !!}
+                                    </div>
+                                    <div class="form-group name">
+                                        {!! Form::label('content', trans('message.totalmoney')) !!}
+                                        {!! Form::text('phone', null, ['class' => 'form-control', 'placeholder' => __('label.property_title')]) !!}
+                                    </div>
+                                    <div class="form-group name">
+                                        {!! Form::label('content', trans('message.note')) !!}<br>
```

Comment:
so sánh 2 cách dùng {{ }} và {!! !!}

Reply:
e thưa chị, khi dùng {!! !!} thì in ra sẽ không kèm theo định dạng html
còn {{ }} sẽ in ra định dang html như <p>abc<p>

Reply:
< p > abc < p >

Reply:
so sánh cái nào hơn cái nào ở chỗ nào ấy
và các trường hợp sử dụng

Reply:
{{ }} khi giá trị truyền vào là biến
{!! !!} khi bên trong có html

Reply:
(facepalm)
chưa nắm được lợi và hại rồi
vì `{!! !!}` sẽ hiển thị cả html bên trong, nên chỉ dùng khi hiển thị description, text area có bao gồm html thôi
còn những nơi không có thì không sử dụng, để không bị hiển thị những chi tiết html ko cần thiết, gây vỡ giao diện nhé
Chỗ này có cần hiển thị html ra ko?